python/bind_operators.h

```diff
@@ -10,6 +10,41 @@
 
 namespace py = pybind11;
 
+template <class T, class... Ignored>
+void bind_common_operators(pybind11::class_<T, Ignored...> &c) {
+  c.def("__repr__", [](T &self) { return to_string(self); });
+  c.def("__bool__", [](T &) {
+    throw std::runtime_error("The truth value of a variable, data array, or "
+                             "dataset is ambiguous. Use any() or all().");
+  });
+  // For views such as VariableView __copy__ needs to return a Variable. Use
+  // result type of operator+ to obtain this type.
+  using Copy = decltype(std::declval<T>() + std::declval<T>());
+  c.def(
```

Comment:
So for `VariableView` it this will return `Variable` but for `Dataset` the return type will still be `Dataset`, since this is the type of the sum of two `Dataset`s? Cool.

Reply:
👍